Dockerfile

```diff
@@ -1,26 +1,13 @@
-FROM alpine:3.7
+FROM google/cloud-sdk:latest
 
-ENV GOOGLE_CLOUD_SDK_VERSION=223.0.0
-ENV GOOGLE_APP_ENGINE_SDK_VERSION=1.9.68
-ENV CLOUDSDK_APP_RUNTIME_ROOT=/google-cloud-sdk/platform/ext-runtime/
-RUN apk add --no-cache curl python
-
-# Install the gcloud SDK
-RUN curl -fsSLo google-cloud-sdk.tar.gz https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GOOGLE_CLOUD_SDK_VERSION-linux-x86_64.tar.gz
-RUN tar -xzf google-cloud-sdk.tar.gz
-RUN rm google-cloud-sdk.tar.gz
-RUN ./google-cloud-sdk/install.sh --quiet
+RUN apt-get install -qqu unzip
 
-# Install the app engine SDKs
-RUN ./google-cloud-sdk/bin/gcloud components install app-engine-go app-engine-java app-engine-php app-engine-python-extras alpha beta
+ENV GOOGLE_APP_ENGINE_SDK_VERSION=1.9.68
```

Comment:
why this version?

Reply:
it's the latest version of the "Original" SDK available: https://cloud.google.com/appengine/docs/standard/go/download#previous-sdk

Reply:
cool, thanks